app/src/main/java/org/sopt/at/navigation/NavItem.kt

```diff
@@ -0,0 +1,6 @@
+package org.sopt.at.navigation
+
+object NavItem {
+    const val MAIN = "main"
+    const val MY = "myscreen"
+}
```

Comment:
```suggestion
sealed class NavItem(val route: String) {
    object Main : NavItem("main")
    object My : NavItem("myscreen")
}
```

sealed class로 관리해서 컴파일 타임 타입 체크로 타입 안정성을 보장하는 방법이 더 보편적이고 안정적인 방법일 것 같아요!